src/MonitoringTargetStage.ts

```diff
@@ -57,21 +90,120 @@ export class MonitoringTargetStack extends Stack {
   }
 
   /**
-	 * Creates a new lambda function and subscribes it to a topic.
-	 *
-	 * This lambda is responsible for formatting events posted to SNS
-	 * and sending notifications to a slack webhook.
-	 *
-	 * @param topic the SNS topic the lambda should be subscribed to
-	 */
+   * Creates a new lambda function and subscribes it to a topic.
+   *
+   * This lambda is responsible for formatting events posted to SNS
+   * and sending notifications to a slack webhook.
+   *
+   * @param topic the SNS topic the lambda should be subscribed to
+   */
   AddLambdaSubscriber(topic: aws_sns.Topic) {
-    const logLambda = new NodejsFunction(this, 'log-lambda', {
+    const monitoringLambda = new MonitoringLambda(this, 'log-lambda');
+    topic.addSubscription(new LambdaSubscription(monitoringLambda.function));
+  }
+
+  private suppressNagging() {
```

Comment:
Is het niet mogelijk NAG van uit de tests te supressen?

Reply:
Mogelijk, maar het voordeel hieraan (als het goed werkt) is dat in je CloudFormation template ingebakken zit waarom je de keuze maakt een regel te negeren. Als je dat alleen in de test doet zit het nergens in de gedeployde resources. Althans, dat is geloof ik de logica die ze zelf volgen ;).